Nos/Views/ProfileView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ProfileView.swift
+//  Nos
+//
+//  Created by Jason Cheatham on 2/20/23.
+//
+
+import Foundation
+import SwiftUI
+struct ProfileView: View {
```

Comment:
Maybe you already know this but there is a more fleshed out ProfileView on `main` you can use.